docs/swagger/teacher/components/schemas/v1/students.request.yaml

```
@@ -0,0 +1,42 @@
+createStudentRequest:
+  type: object
+  properties:
+    lastName:
+      type: string
+      description: 姓
+    firstName:
+      type: string
+      description: 名
+    lastNameKana:
+      type: string
+      description: 姓(かな)
+    firstNameKana:
+      type: string
+      description: 名(かな)
+    mail:
+      type: string
+      description: メールアドレス
+    schoolType:
+      type: integer
+      format: int32
+      description: 校種(0:不明, 1:小学校, 2:中学校, 3:高等学校)
```

API仕様書のリクエスト値では `0:不明` の値は消しといたほうが良さそう
実装見てると、送られてきても400になるだけなので

```suggestion
      description: 校種(1:小学校, 2:中学校, 3:高等学校)
```